Helper/Metric.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Helper;
+
+/**
+ * Boltpay Metrics used by the Metric Client
+ */
+class Metric implements \JsonSerializable
```

Comment:
I wonder if we should define this in MetricsClient.php file. It seems it's kind of implementation detail of MetricsClient